test_runner/fixtures/zenith_fixtures.py

```diff
@@ -637,6 +637,24 @@ def __init__(self, port: int, auth_token: Optional[str] = None) -> None:
     def check_status(self):
         self.get(f"http://localhost:{self.port}/v1/status").raise_for_status()
 
+    def timeline_detail(self, tenant_id: uuid.UUID, timeline_id: uuid.UUID):
```

Comment:
There's `timeline_details` already: https://github.com/zenithdb/zenith/blob/8ab4c8a050895a8e6fe5ba53708d9f3c42e59ed4/test_runner/fixtures/zenith_fixtures.py#L692

so I'd rather reuse that.

Reply:
Good catch! It seems I missed it when it came with rebase on top of you remote storage tests PR :)

Reply:
I've merged both versions together, so it accepts UUID instead of strings like other client methods